### ISSUE TITLE
HITME error on PluginSubCompactor

### ISSUE THREAD

**Issue by MoonbeamLure** [Michael-wigontherun/ESLifyEverything#4]
Hi there. I'm sorry that I seem to keep coming back for more bugs I find, but I am just testing everything I can test lol. Hope you don't mind.

When I ESLify files using the SubPluginCompactor, it will output plugins that are changed. However, sometimes if you do so, xEdit will output a Higher Than Mastery Entry, it seems always on record Tamriel "Skyrim" [WRLD:0000003C] where it will replace the header bits for that record 00 with the load order slot (in this case 0A).

I had never heard of this error, so I went looking up and couldn't find much, but I found this in case you're not sure either: https://dyndolod.info/Messages/Internal-File-FormID-Is-A-HITME

<img width="1337" alt="image" src="https://user-images.githubusercontent.com/106170274/197870565-3efa3849-3d93-4bf1-a949-325ee3ae8500.png">

<img width="1319" alt="image" src="https://user-images.githubusercontent.com/106170274/197871163-46ffafb0-6e35-4557-ab40-cef674121f08.png">

Both of these plugins have a shitload of unresolved FormIDs, but I don't think that'd cause it when it was shifting records around for subplugincompaction.  There were some mods that didn't have errors before and then had errors after.

I've honestly not ever seen this error so I'd have no idea what info you need.

For some of them I'm able to open them up in CK and re-save them and that *seems* to fix it, but others will crash the CK with an access exception (c0000005). 

Neither of the logs (dev or not) presented anything useful such as which mods needed to have forms written to these mods nor which Forms they were, so I can't really help beyond that.

**Comment by Michael-wigontherun** [comment]
Can you search for anything that says exactly this "Removing WorldSpace Cell" in the log with DevLog enabled? If you never see that its not replacing the actual worldspace record and it is indeed a issue in Mutegen.

**Comment by MoonbeamLure** [comment]
> Can you search for anything that says exactly this "Removing WorldSpace Cell" in the log with DevLog enabled? If you never see that its not replacing the actual worldspace record and it is indeed a issue in Mutegen.

Ok. Yeah, nothing about removing a worldspace cell within ESLify Everything's log. Once I get some time I'll submit another issue over at Mutagen's github. Thanks.

**Comment by Michael-wigontherun** [comment]
Noggog is already looking into it.

**Comment by Michael-wigontherun** [comment]
Have you tried to load into the game and can you get to whatever the mod added into the worldspace?

**Comment by Michael-wigontherun** [comment]
I managed to replicate the error. I didn't get a crash when trying to pick up the objects from the mod with the error. I did find a potential fix that output my plugins without the error.
Try this I added a build parameter to the output for the plugins, redownload or delete the overwrite the ESLify Everything output. Then run it over all the mods again.
[https://github.com/Michael-wigontherun/ESLifyEverything/releases/tag/V3.3.2P](https://github.com/Michael-wigontherun/ESLifyEverything/releases/tag/V3.3.2P)
